### DISCUSSION TITLE
How would I disable all features but the debugger? [question]

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [microsoft/vscode-cpptools#6966] [Q&A] [answered]
Hi, so recently Ive switched over to using the LLVM Clangd extension 100%, however I needed to use the debugger with Unreal Engine recently which relies on the debugger coming with this extension.

Is there anyway that I can use only the debugger and disable the rest of the functionality to avoid the 2 language servers stepping on eachother?

**Comment by bobbrow** [comment, marked as answer]
Use the `"C_Cpp.intelliSenseEngine": "Disabled"` setting.
